app/src/main/java/com/stevesoltys/seedvault/transport/backup/PackageService.kt

```diff
@@ -22,13 +23,15 @@ private val TAG = PackageService::class.java.simpleName
 
 private const val LOG_MAX_PACKAGES = 100
 
+// This must be kept in sync with frameworks/base/services/backup/java/com/android/server/backup/utils/BackupEligibilityRules.java
+private const val IGNORE_ALLOW_BACKUP_IN_D2D = 183147249L
```

Comment:
Where does that come from, can we import it?

Reply:
https://cs.android.com/android/platform/superproject/+/master:frameworks/base/services/backup/java/com/android/server/backup/utils/BackupEligibilityRules.java;l=82;drc=dada247f224cfd53ab6b5a56c2bd136badea12f0;bpv=0;bpt=1
It cannot be imported

Reply:
Could we then also add the docs that AOSP has for this that explain what this constant does?
A link to where it comes from would also be nice.